frontend/packages/console-shared/src/hooks/useUserSettings.ts

```diff
@@ -115,27 +65,34 @@ export const useUserSettings = <T>(
               [keyRef.current]: seralizeData(defaultValueRef.current),
             }),
           },
-        });
-        if (!cmCreated) {
-          setSettings(deseralizeData(defaultValueRef.current));
-          setLoaded(true);
-        }
+        })
+          .then(() => setFallbackLocalStorage(false))
+          .catch((err) => {
+            if (err.response.status === 403) {
+              setFallbackLocalStorage(true);
+            } else {
+              setFallbackLocalStorage(false);
+              setSettings(deseralizeData(defaultValueRef.current));
+              setLoaded(true);
+            }
```

Comment:
How will this work at all if we have any error creating the configmap or if the new backend returns an error?
I suppose that it should never error out if the service is working correctly.
Perhaps it shouldn't go to the loaded state if we have an error.
Thoughts?

Reply:
yeah ideally it should never hit this block , have added it if in considering if it errors our and not with forbidden(403) then pass loaded with default value